templates/scripts/convert-ts-to-js.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# This script converts the TypeScript template apps to plain JavaScript, using detype
+# (https://www.npmjs.com/package/detype) to preserve comments and formatting
+# (which tsc does not do)
+
+set -e
+set -o pipefail
+
+convert() {
+  pushd $1
+  # The Babel config causes detype to output non-ES2015 JS, so temporarily hide it
+  mv babel.config.js babel.config.js.ignore
+  # Strip types from every .ts* file (except those in node_modules) and write the output
+  # to the corresponding file in react-native-template-realm-js, with the extension changed
+  # to ".js" (regardless of whether the original was ".tsx" or ".ts")
+  for i in `find . -path ./node_modules -prune -o -name "*.ts*" -print`; do npx detype $i $2/${i/\.ts*/\.js} ; done
```

Comment:
@takameyer I'm wondering if maybe this should only convert files in `app/`, and delete all the `*.ts*` files in there first in case we delete/rename a file... WDYT?

Reply:
That makes sense.  We should probably move the `App.tsx` into `app/` as well?

Reply:
See https://github.com/realm/realm-js/pull/4374/commits/a4917cbaf1f28718287ea7f4b53d831eb5b0f76e